kernel/classes/datatypes/ezobjectrelation/ezobjectrelationtype.php

```diff
@@ -158,23 +158,33 @@ function storeObjectAttribute( $contentObjectAttribute )
         $contentObjectID = $contentObjectAttribute->ContentObjectID;
         $contentObjectVersion = $contentObjectAttribute->Version;
 
-        $obj = $contentObjectAttribute->object();
-        //get eZContentObjectVersion
-        $currVerobj = $obj->version( $contentObjectVersion );
-        // get array of language codes
-        $transList = $currVerobj->translations( false );
-        $countTsl = count( $transList );
+        $contentObject = $contentObjectAttribute->object();
+
+        // check if previous relation(s) can be removed according to existing translations
+        if ( $contentObjectAttribute->contentClassAttributeCanTranslate() )
+        {
+            // get eZContentObjectVersion
```

Comment:
Nitpick: you can typehint the variable in the comment with eZContentObjectVersion it will provide the same info and make it IDE friendly for completion.